payu/models/cice.py

```diff
@@ -45,6 +45,8 @@ def __init__(self, expt, name, config):
 
         self.ice_nml_fname = 'cice_in.nml'
 
+        self.history_nml_fname = 'ice_history.nml'  # only used by payu
```

Comment:
We should probably add this to `self.optional_config_files` so that it is automatically added to the `git` repo. It will also result in the config file being copied to the `work` directory, and archived to `outputXXX`. Arguably this step isn't necessary, as it has already been patched into the main namelist file, but I don't see much harm in it.

Reply:
Its possible for other parts of the payu driver to overwrite settings in `ice_history.nml` (e.g. you can put all of `cice_in.nml` in it, including say restart times ) and as `cice_in.nml` is the one loaded by the model its potentially confusing?

Happy to go either way I think.

Reply:
Fair enough. Let's leave it out and leave it up to config QA testing to take care of these things.